index.js

```diff
@@ -0,0 +1,93 @@
+document.addEventListener("DOMContentLoaded", () => {
+    
+    let teacherInput =  document.querySelector("#teacherInput")
+    let classInput = document.querySelector("#classInput")
+    let allClassesDiv = document.querySelector("#allClasses")
+    let button =  document.querySelector("#allClassesButton") 
+    let collapse = document.querySelector("#collapse")
+    let form =  document.querySelector("#classForm")
+// class search
+    let classSearchDiv = document.querySelector("#studentsByClass")
+    let classSearchForm = document.querySelector("#classSearch")
+    let classSearchInput = document.querySelector("#classSearchInput")
+    let classSearchButton = document.querySelector("#classSearchButton")
+// enroll student elements
+    let enrollStudentsDiv =  document.querySelector("#enrollStudentsDiv")
+    let enrollStudentsForm = document.querySelector("#enrollStudents")
+    let studentNameInput = document.querySelector("#studentNameInput")
+    let studentAgeInput = document.querySelector("#studentAgeInput")
+    let studentClassInput = document.querySelector("#classToEnroll")
+
+    enrollStudentsForm.addEventListener("submit", async (e) => {
+        e.preventDefault();
+        try {
+            let data = await axios.post(`http://localhost:3000/classes/enroll/${studentNameInput.value}/${studentClassInput.value}`)
```

Comment:
In here your axios post should look like `axios.post('url/class/<className>/enroll', {insert here the student object })`